frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -1199,10 +1222,36 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                 return webAnalyticsFilters.some((filter) => filter.key === '$os')
             },
         ],
+        replayFilters: [
+            (s) => [s.webAnalyticsFilters, s.dateFilter, s.shouldFilterTestAccounts],
+            (
+                webAnalyticsFilters: WebAnalyticsPropertyFilters,
+                dateFilter,
+                shouldFilterTestAccounts
+            ): RecordingUniversalFilters => {
+                return {
+                    date_from: dateFilter.dateFrom,
+                    date_to: dateFilter.dateTo,
+                    duration: [
+                        {
+                            type: PropertyFilterType.Recording,
+                            key: 'duration',
```

Comment:
Either will work but active duration might show more interesting recordings
```suggestion
                            key: 'active_duration',
```